### PR TITLE
[Doc] add missing installation steps for the todo plugin

### PR DESCRIPTION
**Pull request by gotson**
Signed-off-by: Gauthier Roebroeck <gauthier.roebroeck@gmail.com>

## Hey, I just made a Pull Request!

The documentation for the `todo` plugin was quite scarce when i tried to install it. This should make it easier for other people to add the plugin!

#### :heavy_check_mark: Checklist

- [ ] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [x] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

**Review by freben — APPROVED** [review]
Nice!

File: plugins/todo-backend/README.md [hunk 1]
```diff
@@ -36,6 +36,19 @@ export default async function createPlugin({
...
+
+```js
+// In packages/backend/src/index.ts
+import todo from './plugins/todo';
+// ...
+async function main() {
+  // ...
+  const todoEnv = useHotMemoize(module, () => createEnv('todo'));
+  // ...
+  apiRouter.use('/todo', await kafka(todoEnv));
```
Comment (julioz) [inline plugins/todo-backend/README.md:49]: should `kafka` be `todo` instead?

File: plugins/todo/README.md [hunk 2]
```diff
@@ -2,6 +2,77 @@
...
+
+```js
+// In packages/backend/src/index.ts
+import todo from './plugins/todo';
+// ...
+async function main() {
+  // ...
+  const todoEnv = useHotMemoize(module, () => createEnv('todo'));
+  // ...
+  apiRouter.use('/todo', await kafka(todoEnv));
```
Comment (julioz) [inline plugins/todo/README.md:59]: same - `kafka` -> `todo`

Comment (gotson) [reply plugins/todo/README.md:59]: Thanks, I opened #8142 to fix it

File: plugins/todo-backend/README.md (hunk 1 above)
Comment (gotson) [reply plugins/todo-backend/README.md:49]: Thanks, I opened #8142 to fix it

**Pull request merged**